### PR TITLE
init development of plgd-cloud helm chart

### PR DESCRIPTION
**Pull request by rafajpet**

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```
Comment (ondrejtomcik) [inline .idea/.gitignore]: please move .gitignore to root folder and ignore whole .idea folder

File: charts/plgd-cloud/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+charts/*
```
Comment (ondrejtomcik) [inline charts/plgd-cloud/.gitignore]: move this to root

**Review by ondrejtomcik — CHANGES_REQUESTED** [review]

File: .idea/.gitignore (hunk 1 above)
Comment (rafajpet) [reply .idea/.gitignore]: It's already in root directory

File: charts/plgd-cloud/.gitignore (hunk 2 above)
Comment (rafajpet) [reply charts/plgd-cloud/.gitignore]: already in root dir

**Review by ondrejtomcik — APPROVED** [review]

**Pull request merged**